.github/workflows/build-liquid.yaml

```diff
@@ -16,21 +16,17 @@ on:
       - "**/README.md"
   workflow_dispatch:
 
+env:
+  CABAL_DIR: '/cabal-dir'  # due to container; see top-level Dockerfile
+
```

Comment:
extra line?

Reply:
Look at the top-level field, like `name` and `on` above: they have empty lines after them. This `env` field simply follows the style.

But I appreciate attention to detail! ❤️

Reply:
One more thing @vidsinghal. You chose to "Request changes" instead of Approve in your review. That's all right. But keep in mind that for trivial changes people usually don't use the "Request Changes" option. Rather, they would approve and left comments. This reduces unnecessary back and forth.

Reply:
ok

Reply:
> Look at the top-level field, like `name` and `on` above: they have empty lines after them. This `env` field simply follows the style.
> 
> But I appreciate attention to detail! ❤️

yeah its sort of inconsistent though

Reply:
Agreed. Making it consistent may be a separate PR. While it's at its current state, we can pick and choose what style to follow. I chose what I feel is a better convention.